sdk/metric/reader.go

```diff
@@ -50,6 +50,8 @@ var errNonPositiveDuration = fmt.Errorf("non-positive duration")
 //
 // Pull-based exporters will typically implement Register
 // themselves, since they read on demand.
+//
+// Warning: methods may be added to this interface in minor releases.
```

Comment:
Shouldn't we add 

> // DO NOT CHANGE: any modification will not be backwards compatible and
	// must never be done outside of a new major release.

To the `Shutdown` method?

We can still add new methods but the existing should not be changed nor removed, right?

Reply:
sure, sounds good